src/scripts/lindprompt

```diff
@@ -45,21 +45,21 @@ while true; do
           then read -r -p "File is not an nexe file. Try to run anyway? [y/N]" response
           case "$response" in
               [yY][eE][sS]|[yY])
-                  $LIND_ROOT/lind/repy/bin/lind ${myarr[1]};
+                  $LIND_ROOT/lind/repy/bin/lind ${myarr[@]:1};
                   continue;
                   ;;
               *)
                   continue;
                   ;;
           esac;
       fi
-      $LIND_ROOT/lind/repy/bin/lind ${myarr[1]};
+      $LIND_ROOT/lind/repy/bin/lind ${myarr[@]:1};
       ;;
     exit|quit)
       exit
       ;;
     *)
-      if result=$(${myarr[@]}); then
+      if result=$(eval ${myarr[@]}); then
```

Comment:
Subshells seem to not work well with pipes. I tried both with and without quotes around it but was unable to get the correct result.

Reply:
Ah that makes sense. Because in the array there is a string representation of a pipe which is not handled correctly by bash, which thinks it's an argument rather than an operator. eval is fine here because it treats everything as text input rather than as arguments.